js/search.js

```diff
@@ -75,14 +75,15 @@ excluded_in_search: true
           "<p>" +
           "<span class='ndx-stats'>Version: " + item.version + "</span>" +
           "<span class='ndx-stats'><a href=\"" + item.pip + "\">PyPI project page</a></span>" +
+        //   "<span class='ndx-stats'><img alt='" + item.name + "' on PyPi' src='" + item.pypi_version_badge + "'></img></span>" +
           "<span class='ndx-stats'><a href=\"" + item.record_url + "\">Record repo</a></span>" +
           "<span class='ndx-stats'>License: " + item.license + "</span>" +
           "</p>" +
           "<p>Maintainers: " + maintainers + "</p>" +
-          "<div class='readme-preview'><p>" + readmeHtml + "</p></div></li>";
+          "<details><div class='readme-preview'><p>" + readmeHtml + "</p></div></details></li>";
         return resultHtml;
       }
-
+https://img.shields.io/pypi/v/ndx-odor-metadata?label=pypi%20package&style=flat-square&color=blue)](https://pypi.org/pypi/ndx-odor-metadata)
```

Comment:
```suggestion

```
I think this snuck in

Reply:
oops, sorry I may have copied and pasted it things in the wrong place.